profiler/profiler.go

```diff
@@ -222,6 +229,9 @@ func (s *Server) DisableProfiler() (string, error) {
 // ProfilerOP will call the doProfilerOP for the ProfilerServer.
 // It will convert the hunam readable op to internal usage and is the entry point for the profiler operations.
 func (c *Client) ProfilerOP(op string, portNumber int32) (string, error) {
+	if !validateOp(op) {
+		return "", fmt.Errorf("invalid operation: %v", op)
+	}
 	return c.doProfilerOP(profilerpb.Op(profilerpb.Op_value[op]), portNumber)
```

Comment:
Is it necessary to have a new function?
```suggestion
        opValue, exists := profilerpb.Op_value[op]
	if !exists {
		return "", fmt.Errorf("invalid operation: %v", op)
	}
	return c.doProfilerOP(profilerpb.Op(opValue), portNumber)
```

Reply:
Nice suggestion! Updated!